docs/reference/mapping/params/term-vector.asciidoc

```diff
@@ -20,9 +22,10 @@ The `term_vector` setting accepts:
 `with_positions`::          Terms and positions are stored.
 `with_offsets`::            Terms and character offsets are stored.
 `with_positions_offsets`::  Terms, positions, and character offsets are stored.
+`with_positions_offsets_payloads`::  Terms, positions, offsets and payloads are stored.
```

Comment:
we also have `with_positions_payloads` , maybe worth documenting that one too?

Reply:
@javanna Thanks, I missed this option. I have added a commit to add this option as well